cl/disclosures/tasks.py

```diff
@@ -372,6 +387,8 @@ def generate_or_download_disclosure_as_pdf(
     :param pdf_url: The URL of PDF in S3
     :return: Response containing PDF
     """
+    logger.info(pdf_url)
+    logger.info(data)
```

Comment:
Are these debug lines, or did you intend them here? If the latter, we should provide more context than just the variable values, and probably put them on one line.

Reply:
these should've been removed.